toolset/setup/linux/prerequisites.sh

```diff
@@ -60,4 +60,7 @@ RETCODE=$(fw_exists ~/.bash_profile.bak)
 
 sudo sh -c "echo '*               -    nofile          65535' >> /etc/security/limits.conf"
 
+#Ensure llvm-dev is installed (required for ruby to install correctly)
+sudo apt-get install llvm-dev
```

Comment:
This can be added to the sudo apt-get above like so: 

```
cloc dstat                        `# Collect resource usage statistics` \
llvm-dev                          `# Needed for ruby`
```